### DISCUSSION TITLE
UnoCSS preset

### DISCUSSION THREAD

**Discussion by prazdevs** [catppuccin/catppuccin#1943] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the requested port be themed?

Port the colors as a preset that extends the theme.

### Has the requested port been themed already?

I ported the colors to unocss as a preset.

### Any additional comments?

Done here https://github.com/prazdevs/unocss-preset-catppuccin 
Not published yet, but build is ready :)

**Comment by catuhana** [comment]
I'd like to mention that there's a maintained port by me too, and it's open to feature requests and such. https://github.com/catuhana/unocss-catppuccin

**Comment by sgoudham** [comment]
Hey @catuhana :wave:,

Would you be interested in submitting your port for review in order for it to be transferred into the Catppuccin organisation?

**Reply by catuhana** [reply]
Hi. I've actually been considering this for a bit, and my goal was to actually do that in the first place, but I never did because I still consider this as *incomplete* - I'm still (thinking of) experimenting/changing stuff, etc. when I have time (and motivation). If that's fine, sure.

**Reply by sgoudham** [reply]
No worries at all, feel free to submit when you have the time/motivation.

I should mention that submitting it for review would help you decide/enforce certain decisions to have consistency with other ports in the organisation.

**Reply by catuhana** [reply]
> I should mention that submitting it for review would help you decide/enforce certain decisions to have consistency with other ports in the organisation.

May I know what the changes I have to make are, other than README changes and such from the porting guide? E.g. will I have to change licence and remove the funding to my personal GitHub for example?

<sup>Sorry if those questions are already answered in docs, I'm not really able to look at them at the moment.</sup>

**Comment by sgoudham** [comment]
Closing as further discussion should take place in https://github.com/catppuccin/catppuccin/issues/2827
